Build failing:
```
======================================================================
ERROR: test_basic_cloudant_seq (corehq.apps.hqadmin.tests.test_utils.TestPillowCheckpointSeqStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/hqadmin/tests/test_utils.py", line 26, in test_basic_cloudant_seq
    pillow_seq_store()
  File "/mnt/corehq/apps/hqadmin/utils.py", line 13, in pillow_seq_store
    db_seq = checkpoint.get_current_sequence_id()
  File "corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 62, in get_current_sequence_id
    return get_or_create_checkpoint(self.checkpoint_id).sequence
AttributeError: 'DocGetOrCreateResult' object has no attribute 'sequence'
```
```
======================================================================
ERROR: test_pillow_reset_checkpoint (pillowtop.tests.test_import_pillows.PillowTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/pillowtop/tests/test_import_pillows.py", line 103, in test_pillow_reset_checkpoint
    pillow.set_checkpoint(Change('123', seq_id))
  File "corehq/ex-submodules/pillowtop/pillow/interface.py", line 57, in set_checkpoint
    self.checkpoint.update_to(change['seq'])
  File "corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 68, in update_to
    checkpoint = self._get_or_create(verify_unchanged=True).document
  File "corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 50, in _get_or_create
    result = get_or_create_checkpoint(self.checkpoint_id)
  File "corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 20, in get_or_create_checkpoint
    checkpoint = DjangoPillowCheckpoint.objects.get(checkpoint_id=checkpoint_id)
  File "/vendor/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 351, in get
    num = len(clone)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 122, in __len__
    self._fetch_all()
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 966, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 265, in iterator
    for row in compiler.results_iter():
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 701, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 787, in execute_sql
    cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
ProgrammingError: relation "pillowtop_djangopillowcheckpoint" does not exist
LINE 1: ...owtop_djangopillowcheckpoint"."old_sequence" FROM "pillowtop...
```

--- a/corehq/ex-submodules/pillowtop/tests/test_checkpoints.py
+++ b/corehq/ex-submodules/pillowtop/tests/test_checkpoints.py
@@ -1,4 +1,4 @@
-from abc import ABCMeta, abstractproperty
+from abc import ABCMeta, abstractproperty, abstractmethod
 from django.test import SimpleTestCase, override_settings, TestCase
 import time
 from dimagi.utils.decorators.memoized import memoized
@@ -25,22 +25,18 @@ class PillowCheckpointTest(SimpleTestCase):
         self.assertEqual(checkpoint_id, PillowCheckpoint(MockDocumentStore(), checkpoint_id).checkpoint_id)
 
 
-class PillowCheckpointDaoTestMixin(object):
+class PillowCheckpointDbTestMixin(object):
     __metaclass__ = ABCMeta
 
     _checkpoint_id = 'test-checkpoint-id'
 
     @abstractproperty
-    def dao(self):
+    def checkpoint(self):
         pass
 
-    @property
-    @memoized
-    def checkpoint(self):
-        return PillowCheckpoint(self.dao, self._checkpoint_id)
-
+    @abstractmethod
     def save_checkpoint(self, checkpoint_id, sequence_id):
-        self.dao.save_document(checkpoint_id, {'seq': sequence_id})
+        pass
 
     def test_get_or_create_empty(self):
         checkpoint_manager = PillowCheckpointManager(MockDocumentStore())
@@ -99,6 +95,22 @@ class PillowCheckpointDaoTestMixin(object):
         self.checkpoint.touch(min_interval=0)
         timestamp_back = self.checkpoint.get_or_create().document['timestamp']
         self.assertNotEqual(timestamp_back, timestamp)
+
+
+class PillowCheckpointDaoTestMixin(PillowCheckpointDbTestMixin):
+    __metaclass__ = ABCMeta
+
+    @abstractproperty
+    def dao(self):
+        pass
+
+    @property
+    @memoized
+    def checkpoint(self):
+        return PillowCheckpoint(self.dao, self._checkpoint_id)
+
+    def save_checkpoint(self, checkpoint_id, sequence_id):
+        self.dao.save_document(checkpoint_id, {'seq': sequence_id})
 
 
 class SimplePillowCheckpointDaoTest(SimpleTestCase, PillowCheckpointDaoTestMixin):

--- a/corehq/ex-submodules/pillowtop/tests/test_checkpoints.py
+++ b/corehq/ex-submodules/pillowtop/tests/test_checkpoints.py
@@ -39,6 +39,9 @@ class PillowCheckpointDaoTestMixin(object):
     def checkpoint(self):
         return PillowCheckpoint(self.dao, self._checkpoint_id)
 
+    def save_checkpoint(self, checkpoint_id, sequence_id):
+        self.dao.save_document(checkpoint_id, {'seq': sequence_id})
+
     def test_get_or_create_empty(self):
         checkpoint_manager = PillowCheckpointManager(MockDocumentStore())
         checkpoint, created = checkpoint_manager.get_or_create_checkpoint('some-id')
@@ -53,7 +56,7 @@ class PillowCheckpointDaoTestMixin(object):
 
     def test_db_changes_returned(self):
         self.checkpoint.get_or_create()
-        self.dao.save_document(self._checkpoint_id, {'seq': '1'})
+        self.save_checkpoint(self._checkpoint_id, '1')
         checkpoint = self.checkpoint.get_or_create().document
         self.assertEqual('1', checkpoint['seq'])
 
@@ -64,7 +67,7 @@ class PillowCheckpointDaoTestMixin(object):
 
     def test_verify_unchanged_fail(self):
         self.checkpoint.get_or_create()
-        self.dao.save_document(self._checkpoint_id, {'seq': '1'})
+        self.save_checkpoint(self._checkpoint_id, '1')
         with self.assertRaises(PillowtopCheckpointReset):
             self.checkpoint.get_or_create(verify_unchanged=True)
 
@@ -76,7 +79,7 @@ class PillowCheckpointDaoTestMixin(object):
 
     def test_update_verify_unchanged_fail(self):
         self.checkpoint.get_or_create()
-        self.dao.save_document(self._checkpoint_id, {'seq': '1'})
+        self.save_checkpoint(self._checkpoint_id, '1')
         with self.assertRaises(PillowtopCheckpointReset):
             self.checkpoint.update_to('2')
 

--- a/corehq/tests/app_test_db_dependencies.yml
+++ b/corehq/tests/app_test_db_dependencies.yml
@@ -128,6 +128,8 @@ corehq.apps.app_manager:
   - custom.ewsghana
   - corehq.apps.smsforms
   - corehq.apps.commtrack
+corehq.apps.change_feed:
+  - pillowtop
 corehq.apps.commtrack:
   - auditcare
   - couchforms

